carma_clock_py/python_wrapper_test.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/python3
+import unittest
+import carma_clock_py
+import time
+
+"""Test Case for testing basic CarmaClock functionality in python
+"""
+class TestCarmaClock(unittest.TestCase):
+
+    def testSimClockInitializeException(self):
+        sim_clock = carma_clock_py.CarmaClock(True)
+        with self.assertRaises(ValueError) as e:
+            sim_clock.nowInMilliseconds()
+        
+        self.assertIn('Clock is not initialized!',str(e.exception))
+
+    def testRealClockTime(self):
+        clock = carma_clock_py.CarmaClock()
+        self.assertAlmostEqual(time.time()*1000, clock.nowInMilliseconds(),delta=10)
+
+    def testSimClockUpdate(self):
+        sim_clock = carma_clock_py.CarmaClock(True)
```

Comment:
Why does this initialization success but case 1 failed in initialization?

Reply:
Please reference the carma-clock library documentation https://github.com/usdot-fhwa-stol/carma-time-lib/blob/develop/include/carma_clock.h